### ISSUE TITLE
Unfocused highlight

### ISSUE THREAD

**Issue by Gubith** [Gruntfuggly/todo-tree#61]
Each Code window properly highlights tags only for the active file.  When using split panel, only one group gets highlighting at a time.  The tags list always shows the correct tags.

**Comment by Gruntfuggly** [comment]
Yes, this is sort of by design, as it is how the extension is coded, but I think it would be nice to be a bit cleverer and keep the highlights for any open files...

**Comment by Gubith** [comment]
Thanks.  A settings toggle would be great.
Changing the color for unfocused helps see where focus is set,  while using the same color helps track where I was looking.   A luminosity offset might work, where -3 for focused color #939393 makes unfocused #909090.

**Comment by Gruntfuggly** [comment]
I've fixed the highlights in all the editors. Changing luminosity is a bit difficult though as there are predefined colours and it has to cope with light and dark themes.

**Comment by CheyiLin** [comment]
Hi @Gruntfuggly thanks for the great extension!
Just found that with latest `0.0.73` the todo highlighting is now working if you switch file tabs.

![todo-tree-highlight-issue](https://user-images.githubusercontent.com/2572321/43435836-5b1df18a-94b4-11e8-94e3-fdeb50097964.gif)

**Comment by CheyiLin** [comment]
Confirmed it has been fixed in  `0.0.74` now.
Thanks for your working!

**Comment by Gruntfuggly** [comment]
Could you check if there is anything in the console log? It works for me, so I think it may be something else is happening before it gets to the highlight.

**Comment by CheyiLin** [comment]
Oh yes forgot to check the console, here is the error when I switch the tab:
```
ERR todoElement is not defined: ReferenceError: todoElement is not defined
	at TodoDataProvider.getElement (C:\Users\user\.vscode\extensions\gruntfuggly.todo-tree-0.0.77\dataProvider.js:520:77)
	at showInTree (C:\Users\user\.vscode\extensions\gruntfuggly.todo-tree-0.0.77\extension.js:662:40)
	at C:\Users\user\.vscode\extensions\gruntfuggly.todo-tree-0.0.77\extension.js:594:17
	at e.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:97:917)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:757:112
	at e.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:97:917)
	at e.$acceptDocumentsAndEditorsDelta (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:755:330)
	at e._doInvokeHandler (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:658:87)
	at e._invokeHandler (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:657:779)
	at e._receiveRequest (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:656:851)
	at e._receiveOneMessage (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:656:620)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:655:544
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:658:673
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:96:599
	at e.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:97:917)
	at u (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:159:721)
	at Socket.<anonymous> (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:159:898)
	at emitOne (events.js:96:13)
	at Socket.emit (events.js:191:7)
	at readableAddChunk (_stream_readable.js:178:18)
	at Socket.Readable.push (_stream_readable.js:136:10)
	at Pipe.onread (net.js:560:20)
```

With my settings:
```json
{
    "todo-tree.highlight": true,
    "todo-tree.flat": true,
    "todo-tree.grouped": true,
    "todo-tree.expanded": true,
    "todo-tree.iconColour": "grey",
    "todo-tree.tags": [
        "BUG",
        "FIXME",
        "TODO",
    ],
    "todo-tree.iconColours": {
        "BUG": "#FF2D00",
        "FIXME": "#F06292",
        "TODO": "#FFBD2A",
    },
    "todo-tree.globs": [
        "!**/vendor/**",
        "!**/node_modules/**",
    ]
}
```

**Comment by Gruntfuggly** [comment]
Thanks - should be fixed now. I hadn't inhibited the file tracking when grouping was enabled - which can't really work because there could be multiple instances of the file in the tree.

**Comment by CheyiLin** [comment]
Cool, confirmed it has been fixed in `0.0.78` now.
Thank you for fixing this so quickly!
